### PR TITLE
Auth type

### PR DESCRIPTION
**Pull request by riteshkarval**

### CODE REVIEW

File: dkube/sdk/rsrcs/operator.py [hunk 1]
```diff
@@ -83,15 +83,20 @@ class DkubeCluster(object):
...
 
     def __init__(self, name=generate("dkube_cluster"), description="", tags=None):
-        self.cluster = Cluster(name=None, kind=None, _class=None, url=None, ca=None, version=None, auth_type=None, jwt_token_type=None, jwt_token=None,
-                               jwt_signing_key=None, cluster_user=None, access_keys=None, tags=None, description=None, plugin=None)
+        self.cluster = Cluster(name=None, kind=None, _class=None, url=None, ca=None, version=None, 
+                               auth_type="none", jwt_token_type=None, jwt_token=None,
+                               jwt_signing_key=None, cluster_user=None, access_keys=None, 
+                               tags=None, description=None, plugin=None)
 
         self.update_basic(name, description, tags)
```
Comment (songole) [inline dkube/sdk/rsrcs/operator.py:88]: don't you need to call all other update_xx methods from here?

Comment (riteshkarval) [reply dkube/sdk/rsrcs/operator.py:88]: The functions update_basic, update_kind, update_url, update_class, update_ca, update_version, update_jwt_details, update_cluster_user, update_plugin, update_access_keys, update_region are there @songole

Comment (songole) [reply dkube/sdk/rsrcs/operator.py:88]: What I meant was, if someone creates DKubeCluster() object with all the parameters specified in the signature, we just use name, description, tags? And, we ignore the rest?

Comment (riteshkarval) [reply dkube/sdk/rsrcs/operator.py:88]: Okay OSM,  I have added the changes.

**Pull request merged**